mlir/include/mlir/Conversion/LuminousToLLVM/LuminousToLLVM.h

```diff
@@ -0,0 +1,23 @@
+//===- LuminousToLLVM.h -----------------------------------------*- C++ -*-===//
```

Comment:
nitpick but this doesn't really fit the style of the rest of the code base. In particularly I feel like the C++ at the end here is misplaced.

Reply:
some .h files have it some don't. for example LinalgToSPIRV.h has it and LinalgToStandard.h doesn't. Every file has a little description though after the file name and I don't. I think if the description is short enough they include the C++. I think in general it is used to not be confused with c header files.

Reply:
I see. kinda strange but if there's precedence for it then that's fine